deployment/docker-compose/template.env

```diff
@@ -74,3 +68,15 @@ LANGFUSE=False
 # LANGFUSE_SECRET_KEY="sk-..."
 # optional based on your Langfuse host
 # LANGFUSE_HOST="https://cloud.langfuse.com"
+
+# Variables for LiteLLM Proxy container #######################################
+# If using OpenAI APIs
+OPENAI_API_KEY="sk-..."
+# If using Vertex AI APIs
+GOOGLE_APPLICATION_CREDENTIALS="/path/to/.gcp-credentials.json"
```

Comment:
Where do we download these from or get these details? Is it there in docs or confluence?

Reply:
I wrote it in the PR description but will add a confluence page

Reply:
Created a confluence page on this!